diff -up octave-4.2.0/m4/acinclude.m4.implicit octave-4.2.0/m4/acinclude.m4
--- octave-4.2.0/m4/acinclude.m4.implicit	2016-11-13 08:16:10.000000000 -0700
+++ octave-4.2.0/m4/acinclude.m4	2016-12-06 14:19:48.866621459 -0700
@@ -1646,12 +1646,13 @@ AC_DEFUN([OCTAVE_CHECK_SIZEOF_FORTRAN_IN
       AC_RUN_IFELSE([AC_LANG_PROGRAM([[
           #include <assert.h>
           #include <stdint.h>
-          ]], [[
           #if defined (OCTAVE_ENABLE_64)
             typedef int64_t octave_idx_type;
           #else
             typedef int octave_idx_type;
           #endif
+          void F77_FUNC(foo,FOO) (octave_idx_type*, octave_idx_type**, octave_idx_type**);
+          ]], [[
           octave_idx_type n = 2;
           octave_idx_type in[2];
           octave_idx_type out[2];
diff -up octave-4.2.0/m4/link.m4.implicit octave-4.2.0/m4/link.m4
--- octave-4.2.0/m4/link.m4.implicit	2016-11-13 08:21:54.000000000 -0700
+++ octave-4.2.0/m4/link.m4	2016-12-06 13:52:47.132003993 -0700
@@ -22,6 +22,7 @@ AC_DEFUN([gl_FUNC_LINK],
        AC_RUN_IFELSE(
          [AC_LANG_PROGRAM(
            [[#include <unistd.h>
+             #include <stdio.h>
            ]],
            [[int result = 0;
              if (!link ("conftest.a", "conftest.b/"))
