Index: vim74/src/po/Makefile
===================================================================
--- vim74.orig/src/po/Makefile
+++ vim74/src/po/Makefile
@@ -21,7 +21,6 @@ LANGUAGES = \
 		ko.UTF-8 \
 		nb \
 		nl \
-		no \
 		pl \
 		pt_BR \
 		ru \
@@ -62,7 +61,6 @@ MOFILES = \
 		ko.UTF-8.mo \
 		nb.mo \
 		nl.mo \
-		no.mo \
 		pl.mo \
 		pt_BR.mo \
 		ru.mo \
@@ -103,7 +101,6 @@ CHECKFILES = \
 		ko.UTF-8.ck \
 		nb.ck \
 		nl.ck \
-		no.ck \
 		pl.ck \
 		pt_BR.ck \
 		ru.ck \
