Index: vim74/runtime/filetype.vim
===================================================================
--- vim74.orig/runtime/filetype.vim
+++ vim74/runtime/filetype.vim
@@ -112,6 +112,10 @@ au BufNewFile,BufRead proftpd.conf*		cal
 " Apache config file
 au BufNewFile,BufRead .htaccess,*/etc/httpd/*.conf		setf apache
 
+" AppArmor
+au BufNewFile,BufRead */etc/apparmor.d/*       setf apparmor
+au BufNewFile,BufRead */etc/apparmor/profiles/*        setf apparmor
+
 " XA65 MOS6510 cross assembler
 au BufNewFile,BufRead *.a65			setf a65
 
