Index: vim-7.4.676.hg.6704/src/configure.in
===================================================================
--- vim-7.4.676.hg.6704.orig/src/configure.in
+++ vim-7.4.676.hg.6704/src/configure.in
@@ -1143,7 +1143,7 @@ eof
 	      if test "${vi_cv_var_python_version}" = "1.4"; then
 		  vi_cv_path_python_plibs="${PYTHON_CONFDIR}/libModules.a ${PYTHON_CONFDIR}/libPython.a ${PYTHON_CONFDIR}/libObjects.a ${PYTHON_CONFDIR}/libParser.a"
 	      else
-		  vi_cv_path_python_plibs="-L${PYTHON_CONFDIR} -lpython${vi_cv_var_python_version}"
+		  vi_cv_path_python_plibs="-lpython${vi_cv_var_python_version}"
 	      fi
 	      dnl -- Check if the path contained in python_LINKFORSHARED is
 	      dnl    usable for vim build. If not, make and try other
